extensions/vercast/src/hooks/use-vercel-info.ts

```diff
@@ -0,0 +1,73 @@
+import { useEffect } from "react";
+import { LocalStorage } from "@raycast/api";
+import { Team, User } from "../types";
+import { fetchUser, fetchTeams } from "../vercel";
+import useSharedState from "./use-shared-state";
+
+const useVercel = () => {
+  /* Establishing state:
+   * user -- used for filtering deployments by user and displaying account information
+   * selectedTeam - used for filtering deployments by team
+   * teams -- used for filtering projects
+   */
+  const [user, setUser] = useSharedState<User>("user");
+  const [teams, setTeams] = useSharedState<Team[]>("teams");
+  const [selectedTeam, setSelectedTeam] = useSharedState<Team>("selectedTeam");
+
+  /*
+   * Populate user, projects, and teams
+   */
+  useEffect(() => {
+    const fetchData = async () => {
+      if (!user || !teams) {
+        const [fetchedUser, fetchedTeams] = await Promise.all([fetchUser(), fetchTeams()]);
+
+        // // add avatar to fetched teams
+        // const fetchedTeamsWithAvatar = await Promise.all(fetchedTeams.map(async (team) => {
```

Comment:
Do we still want this? Or remove?

Reply:
Removed for now, it requires supporting multiple image formats + the API isn't publicly documented